Kconfig.dependencies

```diff
@@ -138,7 +138,12 @@ config SIDEWALK_MFG_STORAGE_DIAGNOSTIC
 config SIDEWALK_STORAGE
 	bool
 	default SIDEWALK
-	imply NVS
+	imply NVS if !SOC_FLASH_NRF_RRAM
```

Comment:
minor: isn't there a config for the traditional flash? It would be more futureproof not tu use a negation in this case

Reply:
Maybe, but this could cause an issue, if both of the configs are enabled, it would be hard to track which one would be selected as backend for Settings. This way, it is clear, that they are mutually exclusive

Reply:
and matter uses the same scheme to select storage backend